StonehearthEditor/pages/scripts/string-property.js

```diff
@@ -520,16 +646,147 @@ RandomBetweenCurvesScalarParameterKind = ParameterKind.extend({
     }),
 });
 
+RandomBetweenCurvesRgbParameterKind = ParameterKind.extend({
+    componentName: 'random-between-curves-rgb-parameter',
+    curveRGB1: null,
+    curveRGB2: null,
+    _onInit: function () {
+        this.set('curveRGB1', CurveRGB.create({}));
+        this.set('curveRGB2', CurveRGB.create({}));
+    }.on('init'),
+    fromJson: function (json) {
+        var curveRGB1 = CurveRGB.create({});
+        curveRGB1.fromJson(json[0]);
+        this.set('curveRGB1', curveRGB1);
+
+        var curveRGB2 = CurveRGB.create({});
+        curveRGB2.fromJson(json[1]);
+        this.set('curveRGB2', curveRGB2);
+    },
+    toJson: function () {
+        return [this.curveRGB1.toJson(), this.curveRGB2.toJson()];
+    },
+    isValid: Ember.computed('curveRGB1.isValid', 'curveRGB2.isValid', function () {
+        return this.curveRGB1.get('isValid') && this.curveRGB2.get('isValid');
+    }),
+});
+
+Point_Burst = Ember.Object.extend({
```

Comment:
Is this a duplicate object? (From emission-property.js)

Reply:
emission was removed, but i forgot to delete the file - done now